modules/administration/src/lib/administration/components/admin-workflow-table/admin-workflow-table.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ButtonModule } from 'primeng/button';
+import { TableModule, TableRowSelectEvent } from 'primeng/table';
+import { Workflow, WorkflowHttpService } from '@te44-front/shared';
+
+@Component({
+  selector: 'app-admin-workflow-table',
+  standalone: true,
+  imports: [CommonModule, ButtonModule, TableModule],
+  templateUrl: './admin-workflow-table.component.html',
+  styleUrl: './admin-workflow-table.component.less',
+})
+export class AdminWorkflowTableComponent implements OnInit {
+  workflows: Workflow[] = [];
```

Comment:
L'utilisation d'un observable ici te permettra d'éviter la gestion de l'unsubscribe du workflowService.getAll()
`workflows$: Observable<Workflow[]> = this.workflowService.getAll();`
en rajoutant 
`*ngIf="workflows$ | async as workflows"` au niveau de la table dans le dom